File: psup assignment/cpp/q3.cpp
#include <iostream>
#include <string>
#include <cctype>
void passwordCheck() {
    std::string password;
    std::cout << "Enter password: ";
    std::cin >> password;
    if (password == "admin123") {
        std::cout << "Welcome" << std::endl;
    }
}
int main() {
    passwordCheck();
    return 0;
}

File: psup assignment/java/q3.java
import java.util.*;
class PasswordCheck {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter password: ");
        String password = input.next();
        if (password.equals("admin123")) {
            System.out.println("Welcome");
        }
        input.close();
    }
}